src/g42-main.cpp

```diff
@@ -304,6 +305,15 @@ int main(int argc, const char **argv) { // the main() function
 	vector<string> argt; // args (without exec name)
 	for (int i=1; i<argc; ++i) argt.push_back(argv[i]);
 	bool early_debug = contains_value(argt, "--d");
+	std::setlocale(LC_ALL, "en_US.UTF-8");
+
+	std::call_once(c_curl_ptr::s_curl_init_flag, []{
```

Comment:
No need for call_once.

* just execute this code (in a code block)

* Add comment please:
// This code MUST be 1-thread and very early in main.

Reply:
seems done (as of 7fa89ee)